src/main/java/seedu/address/logic/commands/appointmentcommands/EditAppointmentCommand.java

```diff
@@ -78,8 +80,15 @@ public CommandResult execute(Model model) throws CommandException {
         Appointment appointmentToEdit = lastShownList.get(index.getZeroBased());
         Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor);
 
-        if (!appointmentToEdit.equals(editedAppointment) && model.hasAppointment(editedAppointment)) {
+        System.out.println(editedAppointment.getName());
```

Comment:
I think you forgot to remove debugging print statement here

Reply:
Removed